### PR TITLE
fix typos, more declarative brod_sock api

### PR DESCRIPTION
**Pull request by juise**

### CODE REVIEW

File: src/brod_group_subscriber.erl [hunk 1]
```diff
@@ -22,7 +22,7 @@
 %%%
 %%% An overview of what it does behind the scene:
 %%%  1. Start a consumer group controller,
-%%%     @see brod_group_controller:start_link/4. to manage the consumer group
+%%%     @see brod_group_controller:start_link/4 to manage the consumer group
```
Comment (id) [inline src/brod_group_subscriber.erl]: The dot here was added intentionally to satisfy Edoc, without dot it fails.

**Comment by juise** [comment]
Ouch!

Do you mind, if i rephrase? - For manage the consumer group state, start a consumer group controller, @see brod_group_controller:start_link/4.

**Comment by id** [comment]
May be even "Start a consumer group controller to manage the consumer group states: @see ..." ?

**Comment by id** [comment]
Very nice, thanks!

**Pull request merged**